docs/source/notebooks/mmm/mmm_example.ipynb

```diff
@@ -1084,296 +1084,296 @@
        "<!-- Generated by graphviz version 8.0.5 (20230430.1635)\n",
```

Comment:
<p>In annext iteration I want to have a similar plot where the x-axis are is true spend (e.g. dollars or EUR) which can bring an absolute view of the contributions per channel.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Reply:
I decided to add it in https://github.com/pymc-labs/pymc-marketing/pull/291/commits/c13112545cb2e4861af6fc6798402046bfdb46e1 😅

Reply:
I decided to add it in https://github.com/pymc-labs/pymc-marketing/pull/291/commits/c13112545cb2e4861af6fc6798402046bfdb46e1 😅